lua_test.go

```diff
@@ -0,0 +1,20 @@
+package lua
+
+import (
+	"regexp"
+	"testing"
+)
+
+// https://github.com/Shopify/go-lua/pull/63
+func TestPushFStringPointer(t *testing.T) {
+	l := NewState()
+	l.PushFString("%p %s", l, "test")
+
+	actual := CheckString(l, -1)
+	ok, err := regexp.MatchString("0x[0-9a-f]+ test", actual)
+	if !ok {
+		t.Error("regex did not match")
+	} else if err != nil {
+		t.Errorf("regex error: %s", err.Error())
+	}
```

Comment:
You could probably accomplish the same without regexp and an exact match doing this:

```go
want := fmt.Sprintf("%p test", l)
if got := CheckString(l, -1); want != got {
	t.Errorf("l.PushFString(`%%p test`, %p) = %q; want %q", l, got, want)
}
```

As an aside, I don't have final say on this, so don't take what I say as requirements. I'm just chiming in because my office uses go-lua in production, so I feel I owe a little bit of my time and effort to watching what's submitted.